node/src/accountant/scanners_tools.rs

```diff
@@ -330,7 +342,7 @@ mod tests {
     }
 
     #[test]
-    fn payable_with_low_debt_is_marked_unqualified() {
+    fn payable_with_debt_smaller_than_permanent_debt_allowed_is_marked_unqualified() {
         let now = SystemTime::now();
         let payment_thresholds = PaymentThresholds::default();
         let unqualified_debt = payment_thresholds.permanent_debt_allowed_gwei - 1;
```

Comment:
Not here but the name of the test `payable_with_low_payout_threshold_is_marked_unqualified` seems cheap. It doesn't say the important thing, that is testing of a debt situated below the declining part of the payment thresholds. It's an important difference from the other tests around here.

Reply:
So, which name seems correct to you?

Reply:
Sure. Still talking about the test that lies below this one. 
I think its name should be like: "debt_under_the_slope_is_marked_unqualified".

As well as another test `payable_above_threshold_values_is_marked_qualified_and_returns_threshold` should say rather:
`debt_above_the_slope_is_qualified_and_the_threshold_point_is_returned`

Reply:
Done!